terratest/src/test/Gopkg.lock

```diff
@@ -0,0 +1,695 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
Should this file be here or should it be removed? (Looks like a lock file that maybe should have been .gitignore'd)

Reply:
I re-added the gopkg.lock file. The sporadic failures with the travis build  is observed after deleting it from git. It appears that Go uses this file for giving reproducible build when changes are moved across the git branches. Similar issue discussed here https://stackoverflow.com/questions/51738797/do-i-need-to-add-gopkg-lock-to-git
Also, checked in few other go repositories, gopkg.lock is preserved in the source code (quick ref: https://github.com/gruntwork-io/terratest).